app/components/primary_nav_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="primary-nav-container">
+  <% nav_items.each do |item| %>
+    <%= item %>
+  <% end %>
+</div>
+<hr class="govuk-!-margin-top-0"/>
```

Comment:
Nice!

Reply:
Do you think you could create unit test for the component as well?

Reply:
@callumacrae you had reservations about component tests. What do you think? @Leercourt too, in case you see this

Reply:
> @callumacrae you had reservations about component tests. What do you think? @Leercourt too, in case you see this

my problem isn't with component tests specifically, my problem is with duplicating tests—or more specifically, duplicating the principles of tests across different types of tests. the component tests in this repo test the rendering of components, which is also what the visual tests will do once they're implemented.

Reply:
Is anything blocking us from adding the visual component tests?

Reply:
nope, the main thing is we just need to convince a github owner to set up the integration for us!

Reply:
@muqi1 is the person I think. Try tagging them on digital tools @callumacrae

Reply:
Going to leave this for now